Updated support for synapse role assignment and Added support for role scope (#14172)
Azure/azure-powershell: src/Synapse/Synapse/ChangeLog.md

        - Additional information about change #1
-->
## Upcoming Release
* Add support for Synapse Role-based access control
   - Upgraded Azure.Analytics.Synapse.AccessControl to 1.0.0-preview.3
   - Updated `New-AzSynapseRoleAssignment` cmdlet
   - Updated `Get-AzSynapseRoleAssignment` cmdlet
   - Updated `Remove-AzSynapseRoleAssignment` cmdlet
   - Added `Get-AzSynapseRoleScope` cmdlet
* Renamed -AllowAllAzureIP to -AllowAllAzureIp and changed IP range to 0.0.0.0-0.0.0.0 
* Added -AllowAllIp and set IP range to 0.0.0.0-255.255.255.255
* Fixed the issue of retrieving Apache Spark pool information through management API

> 

Reviewer: ```suggestion
```

        - Additional information about change #1
-->
## Upcoming Release
* Add support for Synapse Role-based access control

>    * Upgraded Azure.Analytics.Synapse.AccessControl to 1.0.0-preview.3
>    * Updated `New-AzSynapseRoleAssignment` cmdlet
>    * Updated `Get-AzSynapseRoleAssignment` cmdlet
>    * Updated `Remove-AzSynapseRoleAssignment` cmdlet
>    * Added `Get-AzSynapseRoleScope` cmdlet

Reviewer: ```suggestion
   - Upgraded Azure.Analytics.Synapse.AccessControl to 1.0.0-preview.3
   - Updated `New-AzSynapseRoleAssignment` cmdlet
   - Updated `Get-AzSynapseRoleAssignment` cmdlet
   - Updated `Remove-AzSynapseRoleAssignment` cmdlet
   - Added `Get-AzSynapseRoleScope` cmdlet
```

        - Additional information about change #1
-->
## Upcoming Release

> * Upgraded `New-AzSynapseRoleAssignment` cmdlet, `Get-AzSynapseRoleAssignment` cmdlet and `Remove-AzSynapseRoleAssignment` cmdlet
> * Added `Get-AzSynapseRoleScope` cmdlet
> * Upgraded Azure.Analytics.Synapse.AccessControl to 1.0.0-preview.3

Reviewer 2: ```suggestion
* Add support for Synapse Role-based access control
   * Upgraded Azure.Analytics.Synapse.AccessControl to 1.0.0-preview.3
   * Updated `New-AzSynapseRoleAssignment` cmdlet
   * Updated `Get-AzSynapseRoleAssignment` cmdlet
   * Updated `Remove-AzSynapseRoleAssignment` cmdlet
   * Added `Get-AzSynapseRoleScope` cmdlet
```

* Added -AllowAllIp and set IP range to 0.0.0.0-255.255.255.255
* Fixed the issue of retrieving Apache Spark pool information through management API

## Version 0.10.0

> * Updated support for workspace role assignment

Reviewer 2: Add the following line above this line:

```
Upgraded Azure.Analytics.Synapse.AccessControl to 1.0.0-preview.3
```
